src/nodes/Image.tsx

```diff
@@ -110,22 +123,32 @@ export default class Image extends Node {
       draggable: true,
       parseDOM: [
         {
-          tag: "div[class=image]",
+          tag: "div[class=~image]",
```

Comment:
```suggestion
          tag: "div[class~=image]",
```

should be this way around